pkg/app/rds_aurora_mysql.go

```diff
@@ -138,6 +141,44 @@ func (a *RDSAuroraMySQLDB) Install(ctx context.Context, namespace string) error
 	if err := kube.WaitOnDeploymentReady(ctx, a.cli, a.namespace, a.bastionDebugWorkloadName); err != nil {
 		return errors.Wrapf(err, "Failed while waiting for deployment %s to be ready, app=%s", a.bastionDebugWorkloadName, a.name)
 	}
+
+	rdsCli, err := rds.NewClient(ctx, awsConfig, region)
+	if err != nil {
+		return err
+	}
+
+	a.vpcID = os.Getenv("VPC_ID")
+
+	// VPCId is not provided, use Default VPC
+	if a.vpcID == "" {
+		defaultVpc, err := ec2Cli.DescribeDefaultVpc(ctx)
+		if err != nil {
+			return err
+		}
+		if len(defaultVpc.Vpcs) == 0 {
+			return fmt.Errorf("No default VPC found")
+		}
+		a.vpcID = *defaultVpc.Vpcs[0].VpcId
+	}
+
+	// describe subnets in the VPC
+	resp, err := ec2Cli.DescribeSubnets(ctx, a.vpcID)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to describe subnets")
+	}
+
+	// Extract subnet IDs from the response
+	var subnetIDs []string
+	for _, subnet := range resp.Subnets {
+		subnetIDs = append(subnetIDs, *subnet.SubnetId)
+	}
+	// create a subnetgroup with subnets in the VPC
+	subnetGroup, err := rdsCli.CreateDBSubnetGroup(ctx, fmt.Sprintf("%s-subnetgroup", a.name), "kanister-test-subnet-group", subnetIDs)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to create subnet group")
+	}
+	a.dbSubnetGroup = *subnetGroup.DBSubnetGroup.DBSubnetGroupName
+
```

Comment:
Let's extract this newly added code in a new function/method in the same source file and call that from here. This method has become too lengthy.